content/en/docs/concepts/sampling/index.md

```diff
@@ -1,26 +1,26 @@
 ---
 title: Sampling
 description:
-  Learn about sampling, and the different sampling options available in
+  Learn about sampling and the different sampling options available in
   OpenTelemetry.
 weight: 80
 ---
 
-With distributed tracing, you observe requests as they move from one service to
-another in a distributed system. It’s superbly practical for a number of
+With distributed tracing, you can observe requests as they move from one service
+to another in a distributed system. It is superbly practical for a number of
```

Comment:
We are generally okay with contractions, so you could leave this as "It's".

Reply:
reverted, thanks